apps/islamic-website/app/sitemap.ts

```diff
@@ -0,0 +1,75 @@
+import { MetadataRoute } from 'next';
+import { SITE_URL } from '../constants';
+import { SUPPORTED_LOCALES } from '../constants';
+
+export interface SitemapUrl {
+  url: string;
+  lastModified: string;
+}
+
+const staticRoutes = [
+  '/',
+  '/academy',
+  '/build',
+  '/career',
+  '/community-hub',
+  '/contact-us',
+  '/dubai-residents-disclaimer',
+  '/fraud-alert',
+  '/mission',
+  '/news',
+  '/partnerships',
+  '/privacy-policy',
+  '/roadmap',
+  '/scam-alert',
+  '/shariah',
+  '/team',
+  '/validators',
+  '/values',
+  '/wallet',
+  '/whitepaper',
+];
+
+const academyModules = [
+  { moduleCount: 1, lessonsCount: 2, releaseDate: new Date('2023-12-12') },
+  { moduleCount: 2, lessonsCount: 3, releaseDate: new Date('2023-12-19') },
+  { moduleCount: 3, lessonsCount: 3, releaseDate: new Date('2023-12-26') },
+];
```

Comment:
use this array
https://github.com/haqq-network/frontend/blob/dev/libs/islamic-website/academy-page/src/lib/academy-preview-page/academy-preview-page.tsx#L24